azure-pipelines.yml

```diff
@@ -75,81 +100,55 @@ steps:
     inputs:
       sourceFolder: 'apps/teams-test-app/build'
       contents: '**'
       targetFolder: '$(Build.ArtifactStagingDirectory)\teams-test-app'
     displayName: 'Copy Test app to artifacts staging directory'
 
   - task: ArchiveFiles@2
-    condition: and(in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'), in(variables['Build.SourceBranch'], 'refs/heads/develop', 'refs/heads/main'))
+    condition: and(
+      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
+      or(eq(variables['Build.SourceBranch'], 'refs/heads/2.0-preview'), startsWith(variables['Build.SourceBranch'], 'refs/tags/v')))
     inputs:
       rootFolderOrFile: '$(Build.ArtifactStagingDirectory)\teams-test-app'
       includeRootFolder: false
       archiveType: 'zip'
       archiveFile: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
       replaceExistingArchive: true
+    displayName: 'Zip Test app artifacts'
 
   - task: PublishBuildArtifacts@1
-    condition: and(in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'), in(variables['Build.SourceBranch'], 'refs/heads/develop', 'refs/heads/main'))
+    condition: and(
+      in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'),
+      or(eq(variables['Build.SourceBranch'], 'refs/heads/2.0-preview'), startsWith(variables['Build.SourceBranch'], 'refs/tags/v')))
     inputs:
       PathtoPublish: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
       ArtifactName: 'teams-test-app'
     displayName: 'Publish Test app artifacts'
 
-  - task: PublishBuildArtifacts@1
-    condition: and(in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'), ne(variables['Build.Reason'], 'PullRequest'))
-    inputs:
-      PathtoPublish: './common/temp/bundleAnalysis'
-      ArtifactName: '$(bundleArtifactName)'
-    displayName: 'Publish bundle analysis'
-
-  - task: AzureRmWebAppDeployment@4
-    condition: and(in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'), eq(variables['Build.SourceBranch'], 'refs/heads/main'))
-    inputs:
-      ConnectionType: 'AzureRM'
-      azureSubscription: '$(AzureSubscription)'
-      appType: 'webApp'
-      WebAppName: 'musicalsink'
-      package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
-    displayName: 'Deploy to musical sink'
-
-  - task: AzureRmWebAppDeployment@4
-    condition: and(in(variables['Agent.JobStatus'], 'Succeeded', 'SucceededWithIssues'), eq(variables['Build.SourceBranch'], 'refs/heads/develop'))
-    inputs:
-      ConnectionType: 'AzureRM'
-      azureSubscription: '$(AzureSubscription)'
-      appType: 'webApp'
-      WebAppName: 'cloudroll'
-      package: '$(Build.ArtifactStagingDirectory)\teams-test-app\$(Build.BuildId).zip'
-    displayName: 'Deploy to cloud roll'
-
-  - task: ESLint@1
```

Comment:
Why are we removing these tasks

Reply:
We're not, they've just been edited and reordered and the GitHub UI (Or I guess also CodeFlow if you're using that) thinks they've been "deleted"